lib/appium_console.rb

```diff
@@ -9,9 +9,17 @@ def self.warn args
 
 require 'pry'
 
-Pry.send(:define_singleton_method, :pry_load_appium_txt) do |opts={}|
-  verbose = opts.fetch :verbose, false
-  Appium.load_appium_txt file: Dir.pwd + '/appium.txt', verbose: verbose
+Pry.send(:define_singleton_method, :pry_load_appium_txt) do |opts = {}|
+  if ARGV.any?
```

Comment:
`ARGV.any?` doesn't ensure we have at least two elements (`ARGV[0]` & `ARGV[1]`). I think `ARGV.length >= 2` would be better.